content/bn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html

```diff
@@ -0,0 +1,109 @@
+                </p>
+                <p>>A কুবারনেটস ক্লাস্টার দুটি ধরণের সংস্থান নিয়ে গঠিত:
+
+                    <ul>
+                        <li>The <b>Control Plane</b> coordinates the cluster</li>
+                        <li><b>Nodes</b> are the workers that run applications</li>
+                    </ul>
+                </p>
+            </div>
+
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>Summary:</h3>
+                    <ul>
+                        <li>Kubernetes cluster</li>
+                        <li>Minikube</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                    <p><i>
+                         কুবারনেটস হল একটি প্রোডাকশন-গ্রেড, ওপেন-সোর্স প্ল্যাটফর্ম যা কম্পিউটার ক্লাস্টারের মধ্যে এবং জুড়ে অ্যাপ্লিকেশন কন্টেইনারগুলির প্লেসমেন্ট (শিডিউলিং) এবং এক্সিকিউশনকে অর্কেস্ট্রেট করে।
+
+                    </i></p>
+                </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2 style="color: #3771e3;">Cluster Diagram</h2>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_01_cluster.svg"></p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><b>কন্ট্রোল প্লেন ক্লাস্টার পরিচালনার জন্য দায়ী.</b>কন্ট্রোল প্লেন আপনার ক্লাস্টারে সমস্ত ক্রিয়াকলাপ সমন্বয় করে, যেমন অ্যাপ্লিকেশনের সময় নির্ধারণ, অ্যাপ্লিকেশনগুলির পছন্দসই অবস্থা বজায় রাখা, অ্যাপ্লিকেশনগুলিকে স্কেলিং করা এবং নতুন আপডেটগুলি রোল আউট করা.</p>
+                <p><b>একটি নোড হল একটি VM বা একটি শারীরিক কম্পিউটার যা একটি কুবারনেটস ক্লাস্টারে কর্মী মেশিন হিসাবে কাজ করে।</b> প্রতিটি নোডের একটি কুবেলেট থাকে, যা নোড পরিচালনা এবং কুবারনেটস কন্ট্রোল প্লেনের সাথে যোগাযোগের জন্য একটি এজেন্ট। নোডের কনটেইনার ক্রিয়াকলাপ পরিচালনা করার জন্য সরঞ্জাম থাকা উচিত, যেমন কন্টেইনার বা ডকার। একটি Kubernetes ক্লাস্টার যা উত্পাদন ট্র্যাফিক পরিচালনা করে তার ন্যূনতম তিনটি নোড থাকা উচিত কারণ যদি একটি নোড নিচে চলে যায়, তাহলে একটি etcd সদস্য এবং একটি কন্ট্রোল প্লেন ইনস্ট্যান্স উভয়ই হারিয়ে যায় এবং রিডানডেন্সি আপস করা হয়। আপনি আরো কন্ট্রোল প্লেন নোড যোগ করে এই ঝুঁকি কমাতে পারেন।</p>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i>Control Planes manage the cluster and the nodes that are used to host the running applications.</i></p>
```

Comment:
Kindly translate this sentence  "Control Planes manage the cluster and the nodes that are used to host the running applications."